multi/imxrt-multi/imxrt-multi.c

```diff
@@ -168,7 +169,13 @@ static void uart_dispatchMsg(msg_t *msg)
 
 	switch (id) {
 		case id_console:
+#if !ISEMPTY(RTT_CHANNEL_CONSOLE)
+			rtt_handleMsg(msg, RTT_CHANNEL_CONSOLE + id_rtt0);
+#elif !ISEMPTY(UART_CONSOLE_USER)
 			uart_handleMsg(msg, UART_CONSOLE - 1 + id_uart1);
+#else
+			dummyuart_handleMsg(msg);
```

Comment:
As discussed, there must be exactly one console enabled at the time (at least one of them must be enabled)

Reply:
Added check in `config.h` to ensure this condition doesn't happen.